src/main/java/io/cdap/plugin/dlp/Utils.java

```diff
@@ -0,0 +1,217 @@
+package io.cdap.plugin.dlp;
+
+import com.google.privacy.dlp.v2.FieldId;
+import com.google.privacy.dlp.v2.Table;
+import com.google.privacy.dlp.v2.Value;
+import com.google.protobuf.Timestamp;
+import com.google.type.Date;
+import com.google.type.TimeOfDay;
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.schema.Schema;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Helper class for converting to DLP data types
+ */
+public class Utils {
+
+  public static StructuredRecord getStructuredRecordFromTable(Table table, StructuredRecord oldRecord)
+    throws Exception {
+    StructuredRecord.Builder recordBuilder = createBuilderFromStructuredRecord(oldRecord);
+
+    if (table.getRowsCount() == 0) {
+      throw new Exception("DLP returned a table with no rows");
```

Comment:
throw specific exception. Also add contextual information in the error message

Reply:
Done